notebooks/gnn/pyg_hetero_mag_cgs.ipynb

```diff
@@ -0,0 +1,302 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# PyG+cuGraph Heterogeneous MAG Example with cuGraph-Service\n",
+    "# Skip notebook test\n",
+    "\n",
+    "### Requires installation of PyG & cuGraph-Service\n",
+    "#### A cuGraph-Service Server must be running"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import rmm\n",
+    "\n",
+    "rmm.reinitialize(pool_allocator=True,initial_pool_size=5e+9, maximum_pool_size=20e+9)"
+   ]
```

Comment:
We should not create RMM Pool on a client on the service.  The only pool the client process should have is the Pytorch Caching Pool Allocator (enabled by default), having a pool here creates unnecessary memory  contention.

Reply:
resolved